src/pam_hbac.h

```diff
@@ -37,6 +37,16 @@
 
 #define free_const(ptr) free(discard_const(ptr))
 
+#ifndef _pam_overwrite
+#define _pam_overwrite(x)      \
```

Comment:
You don't have to define pam_overwrite again, we alredy have it in pam_hbac_compat.h, because normally it comes from one of the Linux-PAM headers.